eahub/profiles/models.py

```diff
@@ -153,6 +154,17 @@ class OrganisationalAffiliation(enum.Enum):
         WILD_ANIMAL_INITIATIVE: "Wild Animal Initiative"
     }
 
+def create_pretty_property_list(propertyClass, standard_list, other_list):
```

Comment:
Nit: This isn't really "creating" anything (in particular it's not creating a database record), it's just formatting some text. Maybe `pretty_property_list` or `prettify_property_list`?